espresso 4 compile error

Hello, I am a user of espresso. I am trying to install 4.0 and running into the following issue after running cmake; and then make from the build directory:
```
/mnt/d/Desktop/automated_simulations/espresso/src/core/bonded_coulomb_p3m_sr.cpp: In function ‘int bonded_coulomb_p3m_sr_set_params(int, double)’:
/mnt/d/Desktop/automated_simulations/espresso/src/core/bonded_coulomb_p3m_sr.cpp:34:33: error: ‘make_bond_type_exist’ was not declared in this scope
   make_bond_type_exist(bond_type);
                                 ^
/mnt/d/Desktop/automated_simulations/espresso/src/core/bonded_coulomb_p3m_sr.cpp:36:3: error: ‘bonded_ia_params’ was not declared in this scope
   bonded_ia_params[bond_type].p.bonded_coulomb_p3m_sr.q1q2 = q1q2;
   ^
/mnt/d/Desktop/automated_simulations/espresso/src/core/bonded_coulomb_p3m_sr.cpp:37:38: error: ‘BONDED_IA_BONDED_COULOMB_P3M_SR’ was not declared in this scope
   bonded_ia_params[bond_type].type = BONDED_IA_BONDED_COULOMB_P3M_SR;
                                      ^
make[2]: *** [src/core/CMakeFiles/EspressoCore.dir/bonded_coulomb_p3m_sr.cpp.o] Error 1
make[1]: *** [src/core/CMakeFiles/EspressoCore.dir/all] Error 2
make: *** [all] Error 2
```
assistance would be appreciated. I tried searching the bug database. thanks, Nick

--- a/src/core/bonded_interactions/bonded_coulomb_p3m_sr.cpp
+++ b/src/core/bonded_interactions/bonded_coulomb_p3m_sr.cpp
@@ -25,7 +25,7 @@
 #include "bonded_coulomb_p3m_sr.hpp"
 #include "communication.hpp"
 
-#ifdef ELECTROSTATICS
+#ifdef P3M
 
 int bonded_coulomb_p3m_sr_set_params(int bond_type, double q1q2) {
   if (bond_type < 0)

--- a/src/python/espressomd/interactions.pxd
+++ b/src/python/espressomd/interactions.pxd
@@ -519,14 +519,10 @@ cdef extern from "bonded_interactions/thermalized_bond.hpp":
     int thermalized_bond_set_params(int bond_type, double temp_com, double gamma_com, double temp_distance, double gamma_distance, double r_cut)
 cdef extern from "bonded_interactions/bonded_coulomb.hpp":
     int bonded_coulomb_set_params(int bond_type, double prefactor)
-cdef extern from "bonded_interactions/bonded_coulomb_p3m_sr.hpp":
-    int bonded_coulomb_p3m_sr_set_params(int bond_type, double q1q2)
-
 
 cdef extern from "immersed_boundary/ImmersedBoundaries.hpp":
     cppclass ImmersedBoundaries:
         void volume_conservation_set_params(const int bond_type, const int softID, const double kappaV)
-
 
 cdef extern from "immersed_boundary/ibm_triel.hpp":
     int IBM_Triel_SetParams(const int bond_type, const int ind1, const int ind2, const int ind3, const double max, const tElasticLaw elasticLaw, const double k1, const double k2)
@@ -548,6 +544,9 @@ IF ELECTROSTATICS:
     cdef extern from "bonded_interactions/bonded_coulomb.hpp":
         int bonded_coulomb_set_params(int bond_type, double prefactor)
 
+IF P3M:
+    cdef extern from "bonded_interactions/bonded_coulomb_p3m_sr.hpp":
+        int bonded_coulomb_p3m_sr_set_params(int bond_type, double q1q2)
 
 cdef extern from "nonbonded_interactions/nonbonded_interaction_data.hpp":
     int virtual_set_params(int bond_type)

